### DISCUSSION TITLE
New User Creation using Zitadel APIs

### DISCUSSION THREAD

**Discussion by ayshPorter** [zitadel/zitadel#8505] [Q&A] [answered]
Hi all, I am fairly new to Zitadel and wanted to check with you how would I create a new user to any zitadel Organisation.
I see that the API which creates a new user has auth-token in its headers, if my user isn't created yet where would I get the auth token from.
Please help me explain the flow of APIs needed to create a user successfully.

**Comment by hifabienne** [comment, marked as answer]
Hi @ayshPorter 
When you create the first ZITADEL instance (cloud or self hosting), you will always also create the first admin user.
With that user you are able to create new users.

Are you using self hosted or Cloud?
If self hosted you can configure the first user in the steps.yaml
